src/main/java/task/Task.java

```diff
@@ -0,0 +1,28 @@
+package task;
+
+public class Task {
+    protected String description;
+    public boolean isDone;
+    public Task(String description) { //creating new object
```

Comment:
Errrm... that is what a constructor is for.
Please remove that redundant comment.